src/app/models/snake.model.ts

```diff
@@ -2,14 +2,18 @@ import { Direction } from "../enums/direction.enum";
 import { Cell } from "./cell.model";
 import { Grid } from "./grid.model";
 
+const INITIAL_SNAKE_POSITIONS = [[0,7], [0,6], [0,5]];
```

Comment:
It'd be better to have this constant in a constants file

Reply:
Great catch!